README.md

```diff
@@ -1,118 +1,63 @@
-   1. run your application with `docker run -p 8080:8080 test-app` replacing
-      `8080:8080` with the port on which your application listens.
-   1. access your running application and enjoy :).
-
-## Configurations
+Nothing stops you from adding [more buildpacks](https://github.com/orgs/paketo-buildpacks/repositories) during the build process such as [procfile](https://github.com/paketo-buildpacks/procfile), [ca-certificates](https://github.com/paketo-buildpacks/ca-certificates) etc.
+
+### Build without the full builder
+
+The [build-ubi-base](https://github.com/paketo-community/builder-ubi-base) builder [has a list](https://github.com/paketo-community/builder-ubi-base/blob/main/builder.toml)  of buildpacks and an extension participating during build. You can also use the [buildpackless ubi builder](https://github.com/paketo-community/builder-ubi-buildpackless-base) which has no buildpacks or the extension participating during build and include only the ones you want as demonstrated on below example for a Node.js app.
+
+```sh
+pack build test-app \
+   --path ./app-dir \
+   --extension docker.io/paketocommunity/ubi-nodejs-extension \
+   --buildpack paketo-buildpacks/node-engine \
+   --buildpack paketo-buildpacks/npm-install \
+   --buildpack paketo-community/build-plan \
+   --builder paketocommunity/builder-ubi-buildpackless-base
+```
 
 ### Specifying a Node version
 
-ubi only supports the latest version of each Node.js stream
-currently available in the ubi version. At the time of writing
-ubi8 supports the Node.js 16, and 18 streams. For example,
-if the latest Node.js version for the 16.x stream in ubi8 is 16.10.1
-then that is your only option when requesting the Node.js 16.x stream.
-Therefore we suggest that you request the Node.js version such that it
-will accept any version of the stream you want to use with something like
-`~16`.
+UBI only supports the latest version of each Node.js stream currently available in the UBI version. At the time of writing ubi8 supports the Node.js 16, 18, and 20 streams. For example, if the latest Node.js version for the 16.x stream in ubi8 is 16.10.1 then that is your **only option** when requesting the Node.js 16.x stream. Therefore we suggest that you request the Node.js version such that it will accept any version of the stream you want to use with something like `~16`.
 
-To specify the version of the Node that is installed, set the `$BP_NODE_VERSION`
-environment variable at build time either directly (ex. `pack build my-app
---env BP_NODE_VERSION=~16`) or through a [`project.toml`
-file](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
+To specify the version of the Node that is installed, set the `$BP_NODE_VERSION` environment variable at build time either directly (ex. `pack build my-app --env BP_NODE_VERSION=~16`) or through a [`project.toml`file](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md)
```

Comment:
I think we could add here that it can also be set through the package.json, ,npmrc file etc.

Reply:
Added it.